src/libsync/account.h

```diff
@@ -94,6 +94,14 @@ class OWNCLOUDSYNC_EXPORT Account : public QObject
     QString davUser() const;
     void setDavUser(const QString &newDavUser);
 
+    /***
+     * With OC 10 this is the equivalent to the sync root.
+     * With ocis and spaces this will be the default folder containing all spaces.
+     */
+    QString defaultSyncRoot() const;
+    void setDefaultSyncRoot(const QString &syncRoot);
```

Comment:
Should we document that setting an empty string sets `hasDefaultSyncRoot` to false?